e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -521,6 +531,10 @@ func (s *IbcEurekaTestSuite) ICS20TransferERC20TokenfromEthereumToCosmosAndBackT
 			s.Require().Empty(resp.Address)
 
 			relayTxBodyBz = resp.Tx
+
+			s.rustFixtureGenerator.AddFixtureStep("receive_packets", ethereumtypes.RelayerMessages{
+				RelayerTxBody: hex.EncodeToString(relayTxBodyBz),
+			})
```

Comment:
So this generator is only needed to generate all the fixtures into a single file? Why don't we have many fixtures instead similar to sp1 fixtures